### PR TITLE
[Docs] Clarify FAQ entry about name

### PR DESCRIPTION
**Pull request by kripken**

### CODE REVIEW

**Comment by kripken** [comment]
cc @tomayac

File: README.md [hunk 1]
```diff
@@ -918,12 +918,18 @@ fully optimized release build, but it can be useful for local debugging.
...
+Binaryen's name was inspired by *Emscripten*'s: Emscripten's name
+[suggests](https://en.wikipedia.org/wiki/Lisa_the_Iconoclast#Embiggen_and_cromulent)
+it converts something into a **script** - specifically *JavaScript* - and
+Binaryen's suggests it converts something into a **binary** - specifically
+*WebAssembly*. Binaryen began as Emscripten's WebAssembly generation and
+optimization tool, so the name fit as it moved Emscripten from something that
+emitted the text-based format JavaScript (as it did from its early days) to the
+binary format WebAssembly (which it has done since WebAssembly launched).
+
+"Binaryen" is pronounced [in the same manner](http://www.makinggameofthrones.com/production-diary/2011/2/11/official-pronunciation-guide-for-game-of-thrones.html)
```
Comment (tomayac) [inline README.md:930]:
```suggestion
"Binaryen" is pronounced [in the same manner](https://www.makinggameofthrones.com/production-diary/2011/2/11/official-pronunciation-guide-for-game-of-thrones.html#:~:text=Targaryen%20%2D%20AIR%2Deez-,Tar%2DGAIR%2Dee%2Din,-Alliser%20Thorne%20%2D%20AL)
```

Comment (tomayac) [reply README.md:930]: Linking to the secure version on `https:` as a non-nit, and the deep-link to the actual pronunciation as a nit remark.

**Review by tomayac — APPROVED** [review]
LGTM modulo the suggestion. 

Little did I know…

**Pull request merged**